packages/api/api_versioing.md

```diff
@@ -0,0 +1,89 @@
+# Cumulus Api Versioning:
```

Comment:
I think since we have this on the wiki we don't need it here in our API docs. It seems like the wiki or tickets are better places to capture approaches/plans and the README would document the final decision.

Reply:
agreed. will remove